### ISSUE TITLE
MAX - EXP - Mono Audio Volume Not Consistant

### ISSUE THREAD

**Issue by OldFlak** [TheGameCreators/GameGuruRepo#5414]
When you start a level that uses mono audio files it sounds great.
When you run the level again all mono sounds will be about half the volume.

Here be an example:
https://youtu.be/b_J_6wsfcCM

**Comment by bluemeenie195** [comment]
does the same thing on my level too

**Comment by OldFlak** [comment]
btw this happens once you have run any level from any project.

So boot MAX up and run any level, if it has any mono sounds they will play nicely.
When you stop the level and run it again mono sounds will be like half volume.
- OR - Shut MAX down completely, and then start him up again.
Go to any level, even a blank new level, and run it. Then close the level and start another level that does have mono audio, and they will play at half volume.

So in short:
MAX will play mono sounds at correct volume on the first run of a level after Booting MAX up.
After that any levels started will play mono sounds at half volume.

**Comment by LeeBamberTGC** [comment]
@OldFlak I created the most basic level, one object 3D looping a mono sound, and was the same volume after numerous test level plays. I suspect the issue relates to your trigger as I noticed the sound only started when you moved closer to the cars. I have attached my working level, can you attach a stock non-working one, thanks!

[SOUNDVOLUMETEST.zip](https://github.com/TheGameCreators/GameGuruRepo/files/15110301/SOUNDVOLUMETEST.zip)

**Comment by OldFlak** [comment]
Hi @LeeBamberTGC 
Yes the sequence is activated by a zone when the player walks through the doorway.
Would not be surprised if it is my coding tho :)
Will check out that example and report back - when MAX startup issue is sorted :)

Thanks for feedback :)

**Comment by LeeBamberTGC** [comment]
@OldFlak MAX fixed, I look forward to your example level, should be a quick fix once I can reproduce :)

**Comment by OldFlak** [comment]
@LeeBamberTGC example level sent :)

**Comment by LeeBamberTGC** [comment]
@OldFlak Was related to the 3D listener not resetting its position and scale settings, sorted for next build.  Due to complaints, the EXP build will now only be updated once an internal team has confirmed it is stable enough for all those depending on the EXP build. I now update a new branch called the DEVTEST branch, to mark it clearly as a build ONLY for those who know the risks. Let me know if you would like to be part of this risky and potentially upsetting build.

**Comment by OldFlak** [comment]
Hi Lee,
Yes would love to be a risk taker 👍

Regards
Michael
________________________________

**Comment by OldFlak** [comment]
Hi Lee,
Yes would love to be a risk taker 👍

Kind Regards
Michael
________________________________

**Comment by LeeBamberTGC** [comment]
@OldFlak Drop me an email and I will send the password :)

**Comment by OldFlak** [comment]
This issue is sorted - Thanks @LeeBamberTGC
